apps/marketplace/components/Brief/Overview/BriefOverviewSectionList.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import AUlinklist from '@gov.au/link-list/lib/js/react.js'
+
+import styles from './BriefOverviewSectionList.scss'
+
+export class BriefOverviewSectionListComponent extends Component {
+  static propTypes = {
+    links: PropTypes.array.isRequired
+  }
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      showLinks: false
+    }
+  }
+
+  preventDefault = e => {
+    e.preventDefault()
+  }
+
+  toggle = () => {
+    this.setState(prevState => ({
+      showLinks: !prevState.showLinks
+    }))
+  }
+
+  buildLinkListItem = link => {
+    const item = {
+      li: {
+        className: 'linkListItem',
+        'data-complete': link.complete
```

Comment:
is this accessible?  using a screen reader, would you know the item was completed?

Reply:
You're right, this isn't accessible.  Maybe we should use the progress indicator?